includes/class-wc-admin-reports-sync.php

```diff
@@ -452,6 +496,82 @@ public static function customer_lookup_process_batch( $batch_number ) {
 			WC_Admin_Reports_Customers_Data_Store::update_registered_customer( $customer_id );
 		}
 	}
+
+	/**
+	 * Delete customer lookup table rows (in batches).
+	 */
+	public static function customer_lookup_delete_batch_init() {
+		global $wpdb;
+		$batch_size = self::get_batch_size( self::CUSTOMERS_DELETE_BATCH_ACTION );
+		$count      = $wpdb->get_var( "SELECT COUNT(*) FROM {$wpdb->prefix}wc_customer_lookup" );
+
+		if ( 0 === $count ) {
+			return;
+		}
+
+		$num_batches = ceil( $count / $batch_size );
+
+		self::queue_batches( 1, $num_batches, self::CUSTOMERS_DELETE_BATCH_ACTION );
+	}
+
+	/**
+	 * Delete a batch of customers.
+	 *
+	 * @param int $batch_number Batch number to import (essentially a query page number).
+	 * @return void
+	 */
+	public static function customer_lookup_delete_batch( $batch_number ) {
```

Comment:
It looks like `$batch_number` is not used inside this function, so there would be no need to declare it.